### DISCUSSION TITLE
CVE-2021-32640

### DISCUSSION THREAD

**Discussion by kaaax0815** [discordjs/discord.js#5714] [Q&A] [answered]
Is this [Vulnerability](https://github.com/advisories/GHSA-6fc8-4gx4-v693) relevant in Discord.JS?

**Comment by vladfrangu** [comment]
It is one of our dependencies, although I'm pretty sure our semver range should probably handle it automatically for further installs. If you can verify that, that'd be awesome!

**Comment by kyranet** [comment, marked as answer]
No, thankfully it isn't relevant in this library: CVE-2021-32640 is only relevant for WebSocket **servers**, discord.js uses the `ws` module to establish a WebSocket **client** connection.
